integration-auth/src/main/java/org/symphonyoss/integration/authentication/jwt/JwtAuthenticationImpl.java

```diff
@@ -171,12 +171,16 @@ public Long getUserIdFromAuthorizationHeader(String configurationId, String auth
    * token
    */
   public JwtPayload getJwtToken(String configurationId, String authorizationHeader) {
-    if (StringUtils.isEmpty(authorizationHeader) || (!authorizationHeader.startsWith(
-        AUTHORIZATION_HEADER_PREFIX))) {
-      return null;
+    if (!StringUtils.isEmpty(authorizationHeader)) {
```

Comment:
You might use a simple if clause to solve this.

```
if ((StringUtils.isNotEmpty(authorizationHeader) && (authorizationHeader.toUpperCase().startsWith(AUTHORIZATION_HEADER_PREFIX))) {
    String jwt = authorizationHeader.substring(AUTHORIZATION_HEADER_PREFIX.length());
    return parseJwtPayload(configurationId, jwt);
}
```

Reply:
Done.